py/torch_tensorrt/dynamo/conversion/aten_ops_converters.py

```diff
@@ -697,6 +697,32 @@ def aten_ops_clamp(
     )
 
 
+@dynamo_tensorrt_converter(torch.ops.aten.scatter.value)
+def aten_ops_scatter_value(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.select.scatter_value(
+        ctx, target, SourceIR.ATEN, name, args[0], args[1], args[2], args[3]
+    )
+
+
+@dynamo_tensorrt_converter(torch.ops.aten.scatter.src)
+def aten_ops_scatter_src(
+    ctx: ConversionContext,
+    target: Target,
+    args: Tuple[Argument, ...],
+    kwargs: Dict[str, Argument],
+    name: str,
+) -> Union[TRTTensor, Sequence[TRTTensor]]:
+    return impl.select.scatter_src(
+        ctx, target, SourceIR.ATEN, name, args[0], args[1], args[2], args[3]
+    )
```

Comment:
It seems the main difference between `impl.select.scatter_src` and `impl.select.scatter_value` is the casting of the `value` tensor. Could these functions (converters and implementation) be merged to a single function that can either accept `float` or `TRTTensor`, and differentiates accordingly?